README.md

```diff
@@ -1,25 +1,25 @@
 Seaweed File System
 =======
 
-[![Build Status](https://travis-ci.org/chrislusf/weed-fs.svg?branch=master)](https://travis-ci.org/chrislusf/weed-fs)
-[![GoDoc](https://godoc.org/github.com/chrislusf/weed-fs/go?status.svg)](https://godoc.org/github.com/chrislusf/weed-fs/go)
-[![RTD](https://readthedocs.org/projects/weed-fs/badge/?version=latest)](http://weed-fs.readthedocs.org/en/latest/)
+[![Build Status](https://travis-ci.org/chrislusf/seaweedfs.svg?branch=master)](https://travis-ci.org/chrislusf/seaweedfs)
+[![GoDoc](https://godoc.org/github.com/chrislusf/seaweedfs/go?status.svg)](https://godoc.org/github.com/chrislusf/seaweedfs/go)
+[![RTD](https://readthedocs.org/projects/seaweedfs/badge/?version=latest)](http://seaweedfs.readthedocs.org/en/latest/)
```

Comment:
I just realized the /docs directory is the same content as the old wiki on Google Code - it seems like it'd be simpler to just use the GitHub wiki.